From 13dc518ff9d5041dad5db95597bc25a6c4a72754 Mon Sep 17 00:00:00 2001
From: Tengfei Fan <quic_tengfan@quicinc.com>
Date: Tue, 28 May 2024 08:35:47 +0800
Subject: [PATCH] QCLINUX: arm64: dts: qcom: sa8775p-ride: add board id and msm
 id support

Add board id and msm id for SA8775p platform for supporting pick correct
DTB.

Change-Id: Ic9dd244732f74eacc5b496107c926d8be34c6892
Signed-off-by: Tengfei Fan <quic_tengfan@quicinc.com>
Upstream-Status: Pending
---
 arch/arm64/boot/dts/qcom/sa8775p-ride.dts | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/qcom/sa8775p-ride.dts b/arch/arm64/boot/dts/qcom/sa8775p-ride.dts
index d8dabd1859b7..dd47406b1095 100644
--- a/arch/arm64/boot/dts/qcom/sa8775p-ride.dts
+++ b/arch/arm64/boot/dts/qcom/sa8775p-ride.dts
@@ -14,7 +14,8 @@
 / {
 	model = "Qualcomm SA8775P Ride";
 	compatible = "qcom,sa8775p-ride", "qcom,sa8775p", "qcom,qcs9100";
-	qcom,msm-id = <533 0x10000>, <534 0x10000>, <534 0x20000>;
+	qcom,msm-id = <533 0x10000>, <534 0x10000>, <534 0x20000>,
+		      <667 0x10000>, <667 0x20000>;
 	qcom,board-id = <0x19 0>, <0x25 1>;
 
 	aliases {
-- 
2.45.2

